schemas/src/main/java/com/scottlogic/deg/schemas/v3/RuleDeserializer.java

```diff
@@ -13,25 +15,20 @@
     public RuleDTO deserialize(
         JsonParser jsonParser,
         DeserializationContext deserializationContext)
-        throws IOException, JsonProcessingException {
```

Comment:
Why removed?

Reply:
`JsonProcessingException` extends `IOException` - IntelliJ suggests the removal as "there is a more general exception". Suggest we keep this change as it removes the suggestion from IntelliJ and has no functional benefit in its presence.